server/server.js

```diff
@@ -342,6 +346,151 @@ app.post('/message', jsonBodyParser, async (req, res) => {
     }
 });
 
+app.post('/heartbeat', jsonBodyParser, async (req, res) => {
+    log(`got a heartbeat from ${req.body.system_id}, flic_last_seen_secs is ${req.body.flic_last_seen_secs}, flic_last_ping_secs is ${req.body.flic_last_ping_secs}`)
+    let hub = await db.getHubWithSystemId(req.body.system_id)
```

Comment:
It looks like `getHubWithSystemId` could return `null` if there aren't any hubs with the give `system_id`. In that case, I think we should have some error handling here and in the other `app.post` calls to let the caller know what happened. Maybe send back a 400?